### PR TITLE
add constraint to poddisruptionbudget to make minAvailable and maxUnavailable exclusive.

### PR DESCRIPTION
**Pull request by morvencao**
- Add comments for the `podDisruptionBudget` to make it clear that `minAvailable` and `maxUnavailable` are exclusive in single `podDisruptionBudget` definition. See: https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget

### CODE REVIEW

**Comment by ymesika** [comment]
Might not be relevant after https://github.com/istio/istio/pull/11307
All those sync between `istio` and `istio-remote` aren't the right approach imho.

File: install/kubernetes/helm/istio/values.yaml [hunk 1]
```diff
@@ -325,6 +325,8 @@ global:
     #   cpu: 100m
     #   memory: 128Mi
 
+  # Specifies pod budget that reacts to disruptions with a PodDisruptionBudget(PDB).
+  # You can specify only one of maxUnavailable and minAvailable in a single PodDisruptionBudget.
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/values.yaml]: I was a bit confused for the comments and the code here:
1) You are updating the comments for `PodDisruptionBudget` but not adding `PodDisruptionBudget` support.
2) You are updating istio but not istio-remote.

Am I missing anything?

Comment (morvencao) [reply install/kubernetes/helm/istio/values.yaml]: The PR is to fix issue when user specify the `minAvailable` and `maxUnavailable` at the same time in single podDisruptionBudget definition.
And `istio-remote` is to be removed, see: https://github.com/istio/istio/pull/11307

**Comment by ymesika** [comment]
@morvencao You can update the istio-remote too. My comment a FYI that it might be removed soon.

File: install/kubernetes/helm/istio/values.yaml (hunk 1 above)
Comment (morvencao) [reply install/kubernetes/helm/istio/values.yaml]: Done

Comment (gyliu513) [reply install/kubernetes/helm/istio/values.yaml]: The title of this PR `add poddisruptionbudget to istio-remote.` should also be updated.

**Comment by morvencao** [comment]
@sdake +1 to remove PDB, as it adds more constraints for Istio upgrade.

**Comment by sdake** [comment]
@morvencao I belive the consensus is to hardcode the PDB values as in PR: https://github.com/istio/istio/pull/11330

Cheers
-steve

**Comment by morvencao** [comment]
Track hardcode the PDB values as in PR: https://github.com/istio/istio/pull/11330

**Pull request closed without merging**